src/Hint/NewType.hs

```diff
@@ -19,9 +19,12 @@ data A = A {b :: !C} -- newtype A = A {b :: C}
 data A = A Int#
 {-# LANGUAGE UnboxedTuples #-}; data WithAnn x = WithAnn (# Ann, x #)
 data A = A () -- newtype A = A ()
+newtype Foo = Foo Int deriving (Show, Eq) -- newtype Foo = Foo Int deriving newtype (Show, Eq)
```

Comment:
I think `-- @Ignore` as a prefix might make the test pass

Reply:
It's not needed, the tests pass without the `@Ignore` annotation